webpack/assets/javascripts/react_app/components/Settings/__tests__/SettingsActions.test.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
```

Comment:
is that required? /cc @amirfefer ?

Reply:
Changed the tests to use `jest` mocking to mock the API itself instead of mocking axios